phylogenetic/rules/export.smk

```diff
@@ -2,25 +2,36 @@
 This part of the workflow collects the phylogenetic tree and annotations to
 export a Nextstrain dataset.
 
-REQUIRED INPUTS:
-
-    metadata        = data/metadata.tsv
-    tree            = results/tree.nwk
-    branch_lengths  = results/branch_lengths.json
-    node_data       = results/*.json
-
-OUTPUTS:
-
-    auspice_json = auspice/${build_name}.json
-
-    There are optional sidecar JSON files that can be exported as part of the dataset.
-    See Nextstrain's data format docs for more details on sidecar files:
-    https://docs.nextstrain.org/page/reference/data-formats.html
-
-This part of the workflow usually includes the following steps:
-
-    - augur export v2
-    - augur frequencies
-
 See Augur's usage docs for these commands for more details.
 """
+
+rule export:
+    """Exporting data files for for auspice"""
```

Comment:
```suggestion
    """Exporting data files for auspice"""
```

Reply:
Done in [c49d1c7](https://github.com/nextstrain/rabies/pull/3/commits/c49d1c768bb0c321ca9a2ba5b90fa79ac9834c74)